pyrogram/types/user_and_chats/user.py

```diff
@@ -170,7 +171,8 @@ def __init__(
         dc_id: int = None,
         phone_number: str = None,
         photo: "types.ChatPhoto" = None,
-        restrictions: List["types.Restriction"] = None
+        restrictions: List["types.Restriction"] = None,
+        is_gay: bool = None
```

Comment:
This should be refactored to an enum, as we can expect more values to become available over time. Simply classifying users as gay or not gay is too simplistic.

Reply:
Also, we don't want to allow `None` values here. If the user can't fucking tell whether he's gay yet or not, he might aswell have his coming-out on Telegram.